certbot/main.py

```diff
@@ -363,6 +364,33 @@ def _init_le_client(config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
+def register(config, unused_plugins):
+    """Create or modify accounts on the server."""
+
+    # Currently, only --update-registration is implemented. Issue #2446
+    # calls for a fuller register verb, to allow better separation of
+    # account management from obtaining certs.
+    if not config.update_registration:
```

Comment:
So, these functions use `determine_account()` to choose what account the user wants to use, and create one if no account currently exists.

I think it would make sense to simply called `determine_account()` here as the default `register` action, but a possible problem is that if accounts already exist, `determine_account()` tries to choose one instead of creating a new one. It's not totally clear to me what the right behavior of the `register` verb ought to be in the case where an account already exists.

* Create a new account no matter what?
* Ask the user whether to create a new account or not?
* Never create a new account?

In any of these cases, is it reasonable for me to modify `determine_account()` so that it has an additional parameter choosing that behavior instead of its default behavior?